install_dependencies.sh

```diff
@@ -121,7 +122,7 @@ install()
         prep_ubuntu
 
         echo "Installing packages..."
-        apt-get install -y "${UB_LIST[@]}"
+        DEBIAN_FRONTEND=$DEBIAN_FRONTEND apt-get install -y --no-install-recommends "${UB_LIST[@]}"
```

Comment:
```suggestion
        apt-get install -y --no-install-recommends "${UB_LIST[@]}"
```

I think you don't need this.

This is like saying

x = 1
x = x

Reply:
I added this because without this I still get the GUI screen prompt from tzdata.